docs/ui.md

```diff
@@ -37,11 +37,6 @@ To add a custom vendor attribute for an ODBC-based connector, you must modify th
 - connectionResolver.tdr
 - connectionBuilder.js
 
-To add a custom vendor attribute for an ODBC-based connector, you must modify these files:
```

Comment:
Looks like a typo, this should have been "JDBC-based connector", then the files below make sense.

Reply:
Resolved in gh-pages-dev-2021.1